bin/pulsar-daemon

```diff
@@ -157,7 +157,7 @@ start ()
     echo starting $command, logging to $logfile
     echo Note: Set immediateFlush to true in conf/log4j2.yaml will guarantee the logging event is flushing to disk immediately. The default behavior is switched off due to performance considerations.
     pulsar=$PULSAR_HOME/bin/pulsar
-    nohup $pulsar $command "$1" > "$out" 2>&1 < /dev/null &
+    nohup $pulsar $command $1 > "$out" 2>&1 < /dev/null &
```

Comment:
```suggestion
    nohup $pulsar $command "$@" > "$out" 2>&1 < /dev/null &
```

Reply:
using `"$@"` is the correct thing to do.

Another change is also needed. `start "$*"` should be replaced with `start "$@"` on line 219.

Reply:
the solution for restart needs also fixing. I can push changes directly to this PR to help in fixing that.

Reply:
In my local machine, it cannot be started using quotation marks. I need to remove the quotation to make it work.